ding_user_frontend.module

```diff
@@ -282,7 +301,7 @@ function ding_user_frontend_build_menu($path = NULL) {
           if (!empty($children[$item['path']])) {
             // Check that the sub-menu should be display, based on the path.
             $pos = stripos($router_item['href'], $item['href']);
-            if (!empty($pos)) {
+            if ($pos !== FALSE && !$pos) {
```

Comment:
After a longer discussion I agreed that on $pos === 0 should yield the same result.